client/workers/AnalysisWrapper.res

```diff
@@ -1,103 +1,32 @@
 open Prelude
+module Decode = Json_decode_extended
```

Comment:
I'm confused by this.

It seems we're deleting this file and replacing it with FSharpAnalysisWrapper. But why doesn't FSharpAnalysisWrapper have equivalent code to this? Is rollbar and exception handling handled elsewhere?

Reply:
Yes, I'm replacing the OCaml AnalysisWrapper with the F# equivalent.

I'm not sure why the F# analysis wrapper doesn't have "equivalent code" to this - I suspect the OCaml one was built up over time organically, as errors were encountered. Regardless, I think adding "equivalent code" can be saved for another PR - this existing F# analysis path does not go through this error-checking, to my knowledge.

Reply:
OK, probably just built up cruft.